addons/website_sale/i18n/website_sale.pot

```diff
@@ -3572,6 +3572,13 @@ msgstr ""
 msgid "Select a pick-up point"
 msgstr ""
 
+#. module: website_sale
+#. odoo-javascript
+#: code:addons/website_sale/static/src/js/website_sale_delivery.js:0
+#, python-format
+msgid "select to see available Pick-Up Locations"
```

Comment:
Did you kept the position by the export translation wizard? It's supposed to be in alphabetic order (S and s are not the same)

Reply:
now I use the export position, good to know for alphabetic order

Reply:
It's not mandatory but it's the best way to aoid having duplicated message in a po file (which is invalid)